INT-1-Prime-number-identification.cpp

```diff
@@ -0,0 +1,25 @@
+#include<bits/stdc++.h>
+using namespace std;
+void Isprime(int n){
+    if(n<=1){
+    cout<<"Number is not Prime ";
+    return ;
+    }
+    int limit=sqrt(n);
+    for(int i=2;i<=limit;i++)
```

Comment:
Add a comment above the for loop explaining the need of it

Reply:
I have added the comment where it is necessary. please review it